runtime/service/npud/core/Core.cc

```diff
@@ -118,5 +118,51 @@ int Core::destroyNetwork(ContextID contextId, ModelID modelId) const
   return 0;
 }
 
+int Core::createRequest(ContextID contextId, ModelID modelId, RequestID *requestId) const
+{
+  VERBOSE(Core) << "createRequest with " << contextId << ", " << modelId << std::endl;
```

Comment:
I think `Core` is too broad. There is much modules in `ONE`. But it may be okay since `npud` is a separate process. It is not likely to interleave log messages from `onert`.

Reply:
I couldn't catch your concerns.. Do you think this verbose log tag should be defined as a function/feature name? I have defined a verbose log tag by associating them with class name.. I will think more about what you said.

Reply:
> What core? `onert` core? `dbus` core? 

It is `npud` core. As far as I imagine,,,, `executor`, `scheduler`, etc. will be added to this `core` class and the `core` will operate all of this. I will find more specific name for this. :)

Reply:
I thought it would be better to avoid using same name in log tag. But I am not sure that `npud` log messages and `onert` log messages will not be interleaved, I am unsure because I still don't understand what `npud` is for. Is it for managing `npu` (scheduling) ? (I think it is not the role of runtime. It would be better to locate out of `runtime` like `onert-micro`. Or is `npud` a kind of runtime? (Then, I think the name is misnomer).